metrics/metrics-tests/pom.xml

```diff
@@ -35,6 +35,11 @@
             <artifactId>james-server-util</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-testing</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This addition causes 

```
Caused by: org.codehaus.plexus.util.dag.CycleDetectedException: Edge between 'Vertex{label='org.apache.james:james-server-testing:3.6.0-SNAPSHOT'}' and 'Vertex{label='org.apache.james:apache-mailet-api:3.6.0-SNAPSHOT'}' introduces to cycle in the graph org.apache.james:apache-mailet-api:3.6.0-SNAPSHOT --> org.apache.james:apache-james-mailbox-api:3.6.0-SNAPSHOT --> org.apache.james:metrics-tests:3.6.0-SNAPSHOT --> org.apache.james:james-server-testing:3.6.0-SNAPSHOT --> org.apache.james:apache-mailet-api:3.6.0-SNAPSHOT
```

Why did you added this for?

Reply:
Because I had an unstable test in metrics in the last run, I didn't realize that it would introduce a cyclic dependency.I have fixed this by moving the `Unstable` tag to testing-base